tests/integration/test_http_get_all.py

```diff
@@ -0,0 +1,77 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import utils
+
+
+def assert_completed_requests(client: utils.Client, requests_amount):
+    """Check get_all request returns expected completed values."""
+    response = client.fetch_all_requests(per_page=requests_amount, page=1, state="complete")
+    assert response.status == 200
+
+    assert all(item["state"] == "complete" for item in response.data.get("items"))
+    assert 0 <= len(response.data.get("items")) <= requests_amount
+
+
+def assert_no_requests_in_process_state(client: utils.Client, requests_amount, submitted_requests):
+    """Check get_all request returns expected in_progress values."""
+    response = client.fetch_all_requests(per_page=requests_amount, page=1, state="in_progress")
+    assert response.status == 200
+
+    assert all(item["state"] == "in_progress" for item in response.data.get("items"))
+
+    response_ids = set(item.get("id") for item in response.data.get("items"))
+    assert not any(request_id in response_ids for request_id in submitted_requests)
+    assert 0 <= len(response.data.get("items")) <= requests_amount
+
+
+def assert_verbose_correct(item_with_verbose, item_without_verbose, parameters):
+    """Check get_all request with verbose=True can be translated to request with verbose=False."""
+    assert item_with_verbose["id"] == item_without_verbose["id"]
+
+    expected_item_without_verbose = {}
+    for key, value in item_with_verbose.items():
+        if key in parameters:
+            if parameters[key] == "size":
+                expected_item_without_verbose[key] = len(value)
+            elif parameters[key] == "full":
+                expected_item_without_verbose[key] = value
```

Comment:
Let's add an `else` block to avoid typos in the config.

Reply:
Done! Thanks.